packages/ember-auto-import/CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Changelog
+## Release (2023-12-12)
+
+ember-auto-import 2.7.1 (patch)
+
+#### :bug: Bug Fix
+* `ember-auto-import`, `@ef4/test-scenarios`
```

Comment:
@mansona `@ef4/test-scenarios` shouldn't show up here, should it?

Should we fix this PR manually, or do you have to fix something elsewhere and recreate this?

Reply:
I answered in Discord but in case anyone is interested. This is just cosmetic and is something that we do want to fix in release-plan (or really in our fork of lerna-changelog) but it's not going to block the release.

Essentially we would need to implement https://github.com/lerna/lerna-changelog/issues/128 in our fork https://github.com/ef4/lerna-changelog